draft-cwbgp-green-common-energy-efficiency-management.md

```diff
@@ -41,7 +41,7 @@ author:
    fullname: Carlos Pignataro
    organization: North Carolina State University
    country: United States of America
-   email: cpignata@gmail.com
+   email: cpignata@gmail.com, cmpignat@ncsu.edu
```

Comment:
For some reason, a similar change in the past induced an error in the submission API. We had at the time to revert back to one single email. Please pick the @ you prefer but please keep only one. Thanks.